docs/clients/courierimap.txt

```diff
@@ -9,6 +9,11 @@ Installing Courier-IMAP 2.x on FreeBSD:
       cp imapd.dist imapd
       vi imapd
 
+Installing Courier-IMAP on Debian: 
+      apt-get install courier-imap courier-imap-ssl courier-pop courier-pop-ssl courier-authlib-mysql courier-authdaemon
```

Comment:
It might be worth mentioning that `courier-pop` and `courier-pop-ssl` are not necessary unless you want to run a POP3 server (just as `courier-imap` and `courier-imap-ssl` are only required if you want to run an IMAP server), although it's probably quite obvious anyway.